### ISSUE TITLE
threshold levels

### ISSUE THREAD

**Issue by AngristProteomics** [jdrudolph/PluginDependentPeptides#2]
in the original table, there is much useful information as probability values or confidence indices. Why are these values not integrated into the output file? (they 'd be useful for other statistical propose)
thanks for the support
Andrew

**Comment by jdrudolph** [comment]
You are absolutely correct! The current version of the plugin extracts only the most basic information. It's just a matter of implementing the conversion. Which additional columns would you find most valuable to add?

**Comment by AngristProteomics** [comment]
in my opinion, it could be very useful have:
DP Score
Position Probab
DP Positional Prob.
and the Intensity (that I believe is the absolute DP intensity value, and if I'm correct, this is a very useful parameter) 
thank's for your job
Andrea
